artifact/v3/header/token.hpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#ifndef MENDER_ARTIFACT_V3_HEADER_TOKEN_HPP
+#define MENDER_ARTIFACT_V3_HEADER_TOKEN_HPP
+
+
+#include <unordered_map>
+#include <string>
+#include <regex>
+
+#include <common/log.hpp>
+#include <common/expected.hpp>
+
+const std::regex artifact_script_regexp {
+	"scripts/Artifact(Install|Reboot|Rollback|RollbackReboot|Failure)_(Enter|Leave)_[0-9][0-9](_[^[:space:]]+)?",
```

Comment:
Should use `(_\S+)?` at the end instead (this is what [the original](https://github.com/mendersoftware/mender-artifact/blob/master/artifact/scripter.go#L50) uses).

Reply:
It's the same thing, but sure.

Reply:
Oh is it? I thought it was only actual space. Then either is fine.